### ISSUE TITLE
lspace/rspace have confusing names

### ISSUE THREAD

**Issue by bert-github** [w3c/mathml-core#232]
(This is part of the I18n WG review.)

3.2.4 Operator, Fence, Separator or Accent <mo>
https://www.w3.org/TR/mathml-core/#operator-fence-separator-or-accent-mo

From the algorithm in [3.3.1.2 Layout of &lt;mrow&gt;](https://www.w3.org/TR/mathml-core/#layout-of-mrow) it follows that the lspace and rspace attributes of an element add space before, respectively after the element. In a right-to-left formula, that means lspace is on the right and rspace on the left. The names suggest otherwise and when the attributes were created, some 28 years ago, they actually did mean left and right.

To avoid confusion, maybe the spec should explicitly say, in a note, that ‘l’ does *not* mean ‘left’. (There is an image, [figure 10](https://www.w3.org/TR/mathml-core/#figure-box-mrow) that contains the words ‘leading’ and ‘trailing’, but doesn't explicitly link them to lspace and rspace.)

**Comment by dginev** [comment]
"leading" and "rear" could be a good pair of words to map to the l/r letters.

**Comment by bkardell** [comment]
I agree we could add clarity here - tbh I personally even find it confusing that there is not a link that really _defines_ them exactly - in any case, @davidcarlisle can you add such a note?

**Comment by davidcarlisle** [comment]
@bkardell  yes I'll make a simple PR to add this thanks for the ping.

**Comment by bkardell** [comment]
Per the WG meeting today:  It is too late to change the names, but we agree to the suggestion above and @davidcarlisle will make the simple PR and then we'll consider this resolved for now.

**Comment by bert-github** [comment]
The I18n WG agrees that a note solves the issue.

I believe the note has not actually been added yet at this point, but I'll remove the i18n label in anticipation.

**Comment by fred-wang** [comment]
> yes I'll make a simple PR to add this thanks for the ping.

@davidcarlisle Thanks. Can you please open a PR if that's not already done? I can't find any at https://github.com/w3c/mathml-core/pulls
